dip-0027.md

```diff
@@ -142,6 +142,10 @@ Asset Unlock transactions might not be mined for multiple reasons. For example,
 As a way to avoid a catastrophic failure if Platform is compromised, Core will limit credit pool withdrawals for at least the first release of Platform. The withdrawal limits may be re-evaluated and updated periodically as the system matures. To evaluate withdrawal validity, withdrawal amounts from the last 576 blocks should be tallied. The withdrawal should not be mined if:
 
 * It requests more DASH than the credit pool contains
+* The withdrawal would result in more than a 2000 DASH reduction in the credit pool over the 576 block-window
```

Comment:
`over the past 576 blocks` instead?

Reply:
> over the past 576 blocks instead?

that's same wording as below, if so maybe should update both lines:

> * The withdrawal would result in more than a 1000 DASH reduction in the credit pool over the 576 block-window

Reply:
Updated in both places